packages/web-components/src/components/content-block-mixed/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,153 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import markdownContents from '../../../../docs/markdown-contents.md';
+
+# `<dds-content-block-mixed>`
+
+> The "Content Block - Mixed Groups" pattern leverages the Content Block as the overall container of this pattern, and allows only a few opinionated block patterns to be included.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+#### JS
+
+```javascript
+import '../content-block-mixed';
+import '../../image/image';
+import '../../content-block/content-block-heading';
+import '../../content-block/content-block-paragraph';
+import '../../content-group/content-group-heading';
+import '../../content-group-cards/content-group-cards';
+import '../../content-group-cards/content-group-cards-item';
+import '../../content-group-pictograms/content-group-pictograms';
+import '../../content-group-simple/content-group-simple';
+import '../../content-item/content-item-heading';
+import '../../pictogram-item/pictogram-item';
+import '../../cta/card-cta';
+import '../../cta/card-cta-footer';
```

Comment:
These `import`s should be from `@carbon/ibmdotcom-web-components` and  have `.js` extensions.